src/Microsoft.DotNet.Tools.Compiler.Native/IntermediateCompilation/Windows/WindowsLinkStep.cs

```diff
@@ -99,6 +105,13 @@ private void InitializeArgs(NativeCompileSettings config)
 				argsList.Add(Path.Combine(config.IlcPath, lib));
 			}
 
+			// Configuration Based Libs
+			var configLibs = ConfigurationLinkLibMap[config.BuildType];
```

Comment:
The build type is parsed into an enum, so in any situations where an invalid build type was passed, it would be handled higher up.